README.md

```diff
@@ -84,7 +84,7 @@ class BlogController extends Controller
             ->allowedFilters('')
             ->defaultSort('')
             ->allowedSorts('')
-            ->paginate($request->perPage ?? 10);
+            ->paginate(request()->get('perPage',10);
```

Comment:
That's a good one, you can also make it shorter by using only the `request` method: 
```php
request('per_page', 10);
```